core/iwasm/aot/aot_runtime.c

```diff
@@ -5894,4 +6013,26 @@ aot_destroy_global(AOTGlobalInstance *global)
         return;
 
     wasm_runtime_free(global);
-}
\ No newline at end of file
+}
+
+AOTFunctionInstance *
+aot_create_function_empty(const AOTModule *module)
+{
+    AOTFunctionInstance *function = runtime_malloc(
+        sizeof(AOTFunctionInstance) + sizeof(AOTImportFunc), NULL, 0);
```

Comment:
A little confused why need to plus `sizeof(AOTImportFunc)`?

Reply:
I am considering that this would enable host APIs (defined in wasm_export.h) to create an `AOTFunctionInstance` from scratch and pass it to the runtime for execution. The runtime would then use the `u.func_import` field to retrieve some information. However, there might be room for optimization in subsequent pull requests.